packages/serverless/src/gcpfunction/http.ts

```diff
@@ -87,7 +87,7 @@ function _wrapHttpFunction(fn: HttpFunction, wrapOptions: Partial<HttpFunctionWr
       name: `${reqMethod} ${reqUrl}`,
       op: 'gcp.function.http',
       ...traceparentData,
-      metadata: { baggage },
+      metadata: { baggage, source: 'url' },
```

Comment:
Perhaps dumb question, but is parameterization even a thing in this context?

I'm not super familiar with GCP, but it seems from [this](https://cloud.google.com/functions/docs/calling/http) and [this](https://cloud.google.com/functions/docs/writing/http#sample_usage) and [this](https://cloud.google.com/functions/docs/writing/http#parsing_http_requests) like data can only be passed either in the query string or in the request body.

If that's true, then I think we can switch this to `'route'`, because it then isn't high cardinality. (That might not be strictly true, because the `region` part of the URL might vary, but we can just substitute a parameter there if we want.)

Reply:
I mostly just defaulted to url here to stay on the safe side - but I agree with your reasoning, will change to `route`.